silo-core/deploy/input/sonic/S_USDC.e_borrowable_USDC.json

```
@@ -0,0 +1,29 @@
+{
+    "deployer": "0x7461d8c0fDF376c847b651D882DEa4C73fad2e4B",
+    "hookReceiver": "CLONE_IMPLEMENTATION",
+    "hookReceiverImplementation": "GaugeHookReceiver.sol",
+    "daoFee": 1500,
+    "deployerFee": 0,
+    "token0": "wS",
+    "solvencyOracle0": "CHAINLINK_S_USD",
+    "maxLtvOracle0": "NO_ORACLE",
+    "interestRateModel0": "InterestRateModelV2.sol",
+    "interestRateModelConfig0": "stableHighCapV12",
+    "maxLtv0": 7500,
+    "lt0": 8000,
+    "liquidationTargetLtv0": 7800,
+    "liquidationFee0": 650,
+    "flashloanFee0": 0,
+    "callBeforeQuote0": false,
+    "token1": "USDC.e",
+    "solvencyOracle1": "CHAINLINK_USDC.e_USD",
+    "maxLtvOracle1": "NO_ORACLE",
+    "interestRateModel1": "InterestRateModelV2.sol",
+    "interestRateModelConfig1": "bridgeETHv12",
```

Shouldn't it be stableHighCapV12 for USDC and bridgeETHv12 for wS? Please double-check if this is the correct requirement.
```suggestion
    "interestRateModelConfig1": "stableHighCapV12",
```